kubernetes/table_kubernetes_namespace.go

```diff
@@ -21,6 +22,9 @@ func tableKubernetesNamespace(ctx context.Context) *plugin.Table {
 		},
 		List: &plugin.ListConfig{
 			Hydrate: listK8sNamespaces,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "phase", Require: plugin.Optional},
```

Comment:
Can we use resource_version as an optional key qual in the list function?

Reply:
If we are adding the resource_version as optional quals, the API returns an error like `Error: The resourceVersion for the provided list is too old.`